ui/app/templates/components/pki-cert-popup.hbs

```diff
@@ -8,20 +8,16 @@
       </li>
       {{#if item.canRevoke}}
         <li class="action">
-          {{#confirm-action
-            confirmButtonClasses="button is-primary"
-            buttonClasses="link is-destroy"
-            onConfirmAction=(action "delete" item)
-            confirmMessage=(concat "Are you sure you want to revoke " item.id "?")
-            showConfirm=(get this (concat "shouldDelete-" (dot-to-dash item.id)))
-            class=(if (get this (concat "shouldDelete-" (dot-to-dash item.id))) "message is-block is-warning is-outline")
-            containerClasses="message-body is-block"
-            messageClasses="is-block"
-            confirmButtonText="Revoke"
-            data-test-cert-revoke-delete=item.id
-          }}
+          <ConfirmAction
+            @buttonClasses="link is-destroy"
+            @onConfirmAction={{action "delete" item}}
+            @confirmTitle="Revoke this cert?"
+            @confirmMessage="This may affect access to Vault data."
```

Comment:
This could be true, but we really don't have a way to tell what they're using the certs for (whether it's vault authentication or something else).

Reply:
How about something more generic like "Any services using this cert may be affected."?

I'm trying to give them an idea of the consequences, but certs are a tricky one

Reply:
👌